### PR TITLE
add 3d multispecies test case and mesh

### PR DESCRIPTION
**Pull request by matthiasdiener**
Cherry-picked from #19.

Also adds the gmsh-generated mesh instead of a symlink to the lassen FS.

### CODE REVIEW

File: smoke_test_ks_3d/run_params.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+wall_time_scale: 100
+wall_penalty_amount: 0
+wall_material: 2
+
+##############
+# model control
+##############
+nlimit: 1
+use_species_limiter: 2
+use_av: 3
```
Comment (majosm) [inline smoke_test_ks_3d/run_params.yaml]:
```suggestion
use_av: 1
```

File: smoke_test_ks_3d/run_params.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+alpha_sc: 0.01
+s0_sc: -5.0
+kappa_sc: 0.5
+order:  1
```
Comment (majosm) [inline smoke_test_ks_3d/run_params.yaml]:
```suggestion
order:  2
```

**Review by majosm — COMMENTED** [review]
Some changes to match the 2D version that @MTCam pointed out.

File: smoke_test_ks_3d/run_params.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+##############
+wall_time_scale: 100
+wall_penalty_amount: 0
+wall_material: 2
+
+##############
+# model control
+##############
+nlimit: 1
+use_species_limiter: 2
```
Comment (matthiasdiener) [inline smoke_test_ks_3d/run_params.yaml]:
```suggestion
use_species_limiter: 1
```

**Review by anderson2981 — APPROVED** [review]
great.

**Pull request merged**